canonical-websites/www.ubuntu.com
# 1660 desktop government

## Done

* use updated pull-quote is compact pattern

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/desktop/government)

## Screenshots

![image](https://user-images.githubusercontent.com/441217/27076295-b84f5b9e-5024-11e7-9f46-6de9c39cffc2.png)
